frontend/src/components/FormControl/FormLabel/FormLabel.tsx

```diff
@@ -73,18 +81,25 @@ export const FormLabel = ({
         )}
       </Box>
       {description && (
-        <FormLabel.Description>{description}</FormLabel.Description>
+        <FormLabel.Description useMarkdown={useMarkdownForDescription}>
+          {description}
+        </FormLabel.Description>
       )}
     </FormLabel.Label>
   )
 }
 
 FormLabel.Label = ChakraFormLabel
 
+interface FormLabelDescriptionProps extends TextProps {
+  useMarkdown?: boolean
+  children: string
+}
 const FormLabelDescription = ({
   children,
+  useMarkdown,
```

Comment:
should this have a default?

Reply:
add default of `false`, which is the same effect but more explicit